commit 7cbdab2041e378486a74cd9d0ae0b39fb4d1af16
Author: TAMUKI Shoichi <tamuki@linet.gr.jp>
Date:   Wed Sep 5 12:26:22 2012 +0900

    Remove tclIndex due to autogenerated file
    
    auto_mkindex will now run with tclsh instead of wish.

diff --git a/configure.in b/configure.in
index 286e459..dcc8442 100644
--- a/configure.in
+++ b/configure.in
@@ -36,7 +36,7 @@ dnl		- add to -L LDFLAGS
 dnl C_INCLUDE_PATH
 dnl		- add to -I CPPFLAGS
 dnl MSYS	- `yes' if cygnus or Mingw windows environment.
-dnl WISH	- wish
+dnl TCLSH	- tclsh
 dnl LN_S	- ln -s
 dnl tcltk_dep	- make dependencies for tcl/tk interface
 dnl
@@ -59,7 +59,7 @@ AC_CONFIG_HEADERS([config.h interface.h])
 AM_MAINTAINER_MODE
 
 SHELL=${CONFIG_SHELL-"/bin/sh"}
-WISH=${WISH-"wish"}
+TCLSH=${TCLSH-"tclsh"}
 
 dnl to use contains() macro (see autoconf/acinclude.m4)
 CONTAINS_INIT
@@ -2287,7 +2287,7 @@ AC_SUBST(SHCFLAGS)
 AC_SUBST(dynamic_targets)
 AC_SUBST(so)
 AC_SUBST(LN_S)
-AC_SUBST(WISH)
+AC_SUBST(TCLSH)
 AC_SUBST(tcltk_dep)
 AC_SUBST(program_transform_name)
 AC_SUBST(INTERFACE_SRCS)
diff --git a/interface/Makefile.am b/interface/Makefile.am
index 6ce4795..6921a83 100644
--- a/interface/Makefile.am
+++ b/interface/Makefile.am
@@ -240,8 +240,7 @@ EXTRA_DIST = \
 	tkpanel.tcl \
 	browser.tcl \
 	misc.tcl \
-	timidity.pel \
-	tclIndex
+	timidity.pel
 
 .ptcl.tcl:
 	sed -e 's@%TCL_DIR%@$(TCL_DIR)@g' $< > $@
@@ -249,6 +248,8 @@ EXTRA_DIST = \
 	sed -e 's@%BIN_DIR%@$(bindir)@g' $< > $@
 clean:
 	rm -f tkmidity.tcl
+	rm -f tclIndex
+	rm -f timidity.el
 	rm -f *.$(OBJEXT)
 	rm -f *.$(so)
 	rm -f *.res
@@ -259,7 +260,7 @@ clean:
 
 tclIndex: $(TCLSRCS)
 	rm -f tclIndex
-	echo 'auto_mkindex . *.tcl; exit' | $(WISH)
+	echo 'auto_mkindex $(srcdir) *.tcl; exit' | $(TCLSH)
 
 if ENABLE_TCLTK
 install_tk = install.tk
diff --git a/interface/tclIndex b/interface/tclIndex
deleted file mode 100644
index e62cfcf..0000000
--- a/interface/tclIndex
+++ /dev/null
@@ -1,62 +0,0 @@
-# Tcl autoload index file, version 2.0
-# This file is generated by the "auto_mkindex" command
-# and sourced to set up indexing information for one or
-# more commands.  Typically each line is a command that
-# sets an element in the auto_index array, where the
-# element name is the name of a command and the value is
-# a script that loads the command.
-
-set auto_index(InitGlobal) [list source [file join $dir tkpanel.tcl]]
-set auto_index(HandleInput) [list source [file join $dir tkpanel.tcl]]
-set auto_index(MakeShuffleList) [list source [file join $dir tkpanel.tcl]]
-set auto_index(AppendMsg) [list source [file join $dir tkpanel.tcl]]
-set auto_index(ClearMsg) [list source [file join $dir tkpanel.tcl]]
-set auto_index(SelectNumber) [list source [file join $dir tkpanel.tcl]]
-set auto_index(SetTime) [list source [file join $dir tkpanel.tcl]]
-set auto_index(DispButtonPlay) [list source [file join $dir tkpanel.tcl]]
-set auto_index(SetVolume) [list source [file join $dir tkpanel.tcl]]
-set auto_index(WriteMsg) [list source [file join $dir tkpanel.tcl]]
-set auto_index(JumpCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(VolumeCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(LoadCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(PlayCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(PauseCheck) [list source [file join $dir tkpanel.tcl]]
-set auto_index(PauseCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(StopCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(QuitCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(PrevCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(NextCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(ForwardCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(BackwardCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(VolUpCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(VolDownCmd) [list source [file join $dir tkpanel.tcl]]
-set auto_index(DispTables) [list source [file join $dir tkpanel.tcl]]
-set auto_index(SaveConfig) [list source [file join $dir tkpanel.tcl]]
-set auto_index(LoadConfig) [list source [file join $dir tkpanel.tcl]]
-set auto_index(InitCmdLine) [list source [file join $dir tkpanel.tcl]]
-set auto_index(SelectList) [list source [file join $dir tkpanel.tcl]]
-set auto_index(OpenFiles) [list source [file join $dir tkpanel.tcl]]
-set auto_index(CloseFiles) [list source [file join $dir tkpanel.tcl]]
-set auto_index(ToggleCurFileMode) [list source [file join $dir tkpanel.tcl]]
-set auto_index(CreateWindow) [list source [file join $dir tkpanel.tcl]]
-set auto_index(filebrowser) [list source [file join $dir browser.tcl]]
-set auto_index(fs:init) [list source [file join $dir browser.tcl]]
-set auto_index(fs:init-lbox) [list source [file join $dir browser.tcl]]
-set auto_index(fs:get-cur-lbox) [list source [file join $dir browser.tcl]]
-set auto_index(fs:select) [list source [file join $dir browser.tcl]]
-set auto_index(fs:selall) [list source [file join $dir browser.tcl]]
-set auto_index(fs:updir) [list source [file join $dir browser.tcl]]
-set auto_index(fs:changedir) [list source [file join $dir browser.tcl]]
-set auto_index(fs:update) [list source [file join $dir browser.tcl]]
-set auto_index(retrieve-filename) [list source [file join $dir misc.tcl]]
-set auto_index(sec2time) [list source [file join $dir misc.tcl]]
-set auto_index(numeric-bind) [list source [file join $dir misc.tcl]]
-set auto_index(my-listbox) [list source [file join $dir misc.tcl]]
-set auto_index(my-dialog) [list source [file join $dir misc.tcl]]
-set auto_index(warning) [list source [file join $dir misc.tcl]]
-set auto_index(error) [list source [file join $dir misc.tcl]]
-set auto_index(information) [list source [file join $dir misc.tcl]]
-set auto_index(question) [list source [file join $dir misc.tcl]]
-set auto_index(rootname) [list source [file join $dir misc.tcl]]
-set auto_index(my-random) [list source [file join $dir misc.tcl]]
-set auto_index(init-random) [list source [file join $dir misc.tcl]]
